packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -710,13 +710,13 @@ const CFDDashboard = (props: TCFDDashboardProps) => {
                                     className='cfd-dashboard__maintenance-icon'
                                 />
                                 <div className='cfd-dashboard__maintenance-text'>
-                                    {platform === CFD_PLATFORMS.DXTRADE ||
+                                    {platform === CFD_PLATFORMS.DXTRADE && (
+                                        <Localize i18n_default_text='Server maintenance starts at 06:00 GMT every Sunday and may last up to 2 hours. You may experience service disruption during this time.' />
+                                    )}
+                                    {platform === CFD_PLATFORMS.MT5 ||
                                         (platform === CFD_PLATFORMS.DERIVEZ && (
-                                            <Localize i18n_default_text='Server maintenance starts at 06:00 GMT every Sunday and may last up to 2 hours. You may experience service disruption during this time.' />
+                                            <Localize i18n_default_text='Server maintenance starts at 01:00 GMT every Sunday, and this process may take up to 2 hours to complete. Service may be disrupted during this time.' />
```

Comment:
Why do we need these changes on cfd-dashboard?

Reply:
I think the changes should only be in `trade-modal.tsx`.
Can you rectify it?

Reply:
@shontzu-deriv You are changing maintenance message of cfd dashboard (which we do not use anymore) while we need to change the one that is in the our "trade_modal"